web/src/main/java/nl/tehdreamteam/se42/web/token/TokenRegistryFactory.java

```diff
@@ -0,0 +1,30 @@
+package nl.tehdreamteam.se42.web.token;
+
+import nl.tehdreamteam.se42.web.token.impl.HashTokenRegistry;
+
+/**
+ * Holds a single instance of a {@link TokenRegistry}.
+ */
+public class TokenRegistryFactory {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1118)